### ISSUE TITLE
They grow up so quick (can't complete)

### ISSUE THREAD

**Issue by remul111** [dalaranwow/dalaran-wow#5318]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I have all the quest items needed to complete the quest - venomhide hatching, 20 venomhide baby tooth, 20 runecloth, 20 rugged leather.  They are all in my characters inventory.  I cannot complete the quest as the continue button is grayed out.

**Expected behaviour**: Tell us what should happen instead
At this point all required items are in my inventory (visually checked) and the quest should be complete.

**Steps to reproduce the problem**:

1. have all items in inventory
2. complete all prior quests up to the they grow up so fast quest
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Yes you can fix that issue by yourself. Just remove the quest from your questlog. (Don't worry the items will still be in your bag). Then reaccept the quest, and then you can directly complete it. Thank you.
